kubetools/dev/backends/docker_compose/config.py

```diff
@@ -123,11 +123,13 @@ def _create_compose_service(kubetools_config, name, config, envvars=None):
 
     service.update(config)
 
-    # Translate k8s command/args to docker-compose entrypoint/command
-    if 'command' in service:
-        service['entrypoint'] = service.pop('command')
-    if 'args' in config:
-        service['command'] = service.pop('args')
+    if 'args' in service:
+        if 'command' in service:
```

Comment:
There is an assumption here that args without command means entrypoint in docker file.

I did wonder whether args with command is a reasonable state to be in? I guess something like a command in main block and additional args in `dev` block might make sense

Reply:
yes, args + command is a valid situation.

In k8s specification, `command` is the ability to override Docker image's `entrypoint` and `args` is docker-compose's `command`

It makes things a little confusing hence this code that was making sure `kubetools.yml` is written with the k8s convention and translated to the docker-compose convention for development

Reply:
There's also a case of keeping separate the semantics of the main command of the container and the specific arguments to apply to the use-case.
Within kubetools context that could also mean a `command` in the containerContext (i.e. applying to all containers using it) and different `args` in each container that uses that context